src/react/ui-elements/EditableKeyValue.jsx

```diff
@@ -0,0 +1,149 @@
+// @flow
+import { Button } from '@scality/core-ui';
+import Input from './Input';
+import React from 'react';
+import { isEmptyItem } from '../utils';
+import styled from 'styled-components';
+
+export const Items = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const Item = styled.div`
+    display:flex;
+    flex-direction:row;
+    align-items: center;
+
+    margin-bottom: 5px;
+
+    // For Select we ajust width here becase in CoreUI.Select
+    // props are not applied to the first parent element.
```

Comment:
It looks like just the UI funkiness but wanted to double-check if it could be that comments are not allowed in here or something like that - probably it's good as it's compiling and passing the CI and all that but wouldn't hurt to double-check I guess.

Reply:
Yes, better doudle checking, it is fine.

Reply:
Cool, probably it was an issue when they are in the render method or something like that, not too too sure tho. :)